src/test/java/com/eppo/sdk/EppoClientTest.java

```diff
@@ -288,8 +288,8 @@ public void testBanditBanditAction() {
 
     // Attempt to get a bandit assignment
     Optional<String> stringAssignment = EppoClient.getInstance().getStringAssignment(
-      "subject1",
-      "test_bandit_1",
+      "subject2",
+      "cold-start-bandit-experiment",
```

Comment:
Changing the experiment name meant I needed a new subject key to get the same assignment. Our per-experiment randomization works 😅